org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ZipActionImpl.java

```diff
@@ -418,6 +433,25 @@ private void applyZipStream(
 		}
 	}
 
+	private void printZipActionDuration(final String inputName) {
+		if (getLogger().isInfoEnabled()) {
+			getLogger().info("Stop processing [ {} ] using [ {} ] took [ {}ms ]: {}",
+							 inputName, getName(),
+							 getActiveChanges().getElapsedMillis(), getActiveChanges().getChangeText());
+		}
+	}
+
+	private void printAdviseOnUnchanged(final String inputName) {
+		if (getLogger().isInfoEnabled() && !getActiveChanges().isChanged()) {
+			// we could at least test all ZipActionImpl, but if we do it
+			// accurately for archive actions (JAR, WAR, EAR, ...), it will dramatically speed up the conversion
+			// kinda compromise
+			getLogger().warn("[ {} ] has been processed without changes. If you don't alter the rules, you can exclude " +
```

Comment:
I don't think this should be a warning. The test above is info enabled. A warning implies a potential problem for which corrective action can be taken. Here it is just a hint that you could avoid work.